### ISSUE TITLE
[Rodan] Workflows are not loading

### ISSUE THREAD

**Issue by martha-thomae** [DDMAL/Rodan#1046]
When I click on any workflow (in any project), I cannot actually see the workflow and I have a 403 error:

  <img width="1228" alt="Screenshot 2023-08-05 at 14 30 59" src="https://github.com/DDMAL/Rodan/assets/13948831/747859c8-7c8e-4e9b-8dad-56919f660476">

---
Also, my projects weren't loading (I could only see four out of more than 20 that I have). But they have been slowly appearing, so I will wait on that.

**Comment by martha-thomae** [comment]
Even when I create a new project, and then within that a new workflow, when I try to open the new (empty) workflow, the same thing happens:

<img width="1229" alt="Screenshot 2023-08-05 at 14 47 17" src="https://github.com/DDMAL/Rodan/assets/13948831/b3c03dd1-eda6-4baa-8611-2ccbc5e81eae">

**Comment by sabrina0822** [comment]
I just checked the database and all of your data is still there, going to spend some time looking into this today but I'm unable to re-create it on my account.

**Comment by timothydereuse** [comment]
i was also unable to recreate it. Martha, are you able to make a new workflow (just add some jobs, link them up, whatever), leave Rodan and then come back and load it again? does that work for you or is it just workflows you already had?

**Comment by martha-thomae** [comment]
Hi! It seems that as soon as I tried to do the same in incognito mode, the issue with the workflows not loading got resolved. Thank you!

(I have another issue, the projects loading kind of weird, but I will include this as a separate thing)
